T/Torch/build_tarballs.jl

```diff
@@ -15,17 +15,18 @@ sources = [
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
+mv libtorch cudnn $prefix
 cd /usr/local/
```

Comment:
you can do `mkdir -p` to avoid having to `cd` all the time. or just pass it an absolute path.

Reply:
Done